examples/psr16-example.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+require "vendor/autoload.php";
```

Comment:
What does this do?

Reply:
The `declare(strict_types=1)` makes PHP act like a strictly typed language and throw an exception when you try to use, for example, a string where an int is required. Without that, PHP will try to cast the value to the required type instead. The `require` loads the autogenerated code the PHP autoloader needs.